pbm/bsontypes.go

```diff
@@ -38,6 +38,9 @@ type NodeInfo struct {
 	Secondary                    bool                 `bson:"secondary,omitempty"`
 	Hidden                       bool                 `bson:"hidden,omitempty"`
 	Passive                      bool                 `bson:"passive,omitempty"`
+	ArbiterOnly                  bool                 `bson:"arbiterOnly"`
+	SlaveDelay                   int                  `bson:"slaveDelay"`
```

Comment:
Let's rename at least our variable to "SecondaryDealy"
Here and in `RSMember struct`